setup.py

```diff
@@ -14,7 +14,7 @@
     license='MIT',
     description='SendGrid Backend for Django',
     long_description=open('./README.rst').read(),
-    install_requires=["sendgrid >= 3.5, < 4"],
+    install_requires=["sendgrid >= 3.5, < 4.0.4"],
```

Comment:
Yes the latest version of the python-sendgrid package is version 4.0.0. It was unnecessary that I changed it to 4.0.4. I do not remember why I did it.

```
$ Pip freeze | Grep -i sendgrid
Sendgrid == 4.0.0
-e git + https: //github.com/lucassimon/sendgrid-django.git@8df68d527f1783392dca7e84bbf04606120c4ac7#egg=sendgrid_django
```

And if so, why do not we fix the whole range of versions?

What are the other versions that should be changed? The same dependency is only from python-sendgrid and changed the main [version](https://github.com/elbuo8/sendgrid-django/pull/53/files#diff-6f46426f25fe107cda3f1185b7c759ee) of this package: